### DISCUSSION TITLE
REST API available?

### DISCUSSION THREAD

**Discussion by bartclaeys** [Myndex/SAPC-APCA#23] [Q&A] [answered]
Is there a REST API available, similar to:

https://webaim.org/resources/linkcontrastchecker/?fcolor=FFFFFF&bcolor=FF0000&lcolor=FFFFFF&api

This would allow integration instead of scraping the APCA page.

**Comment by Myndex** [comment]
Hi @bartclaeys  thank you for commenting,

I can add that to the dev list, and I have considered it. Right now working on a few other pressing things including setting up npm.

Because APCA operates well as plain vanilla JS and is very easy to integrate, I hadn't placed a high priority on REST, in particular due to concerns about traffic/hits etc.

The only small issue is right now in the repo I'm only posting the base code, and not the look up tables, as they are under revision. They will soon be replaced with a parametric method, and then I'll have a drop in module which is more ideal for performance.

Thank you,

Andy

**Comment by Myndex** [comment, marked as answer]
There is an npm package available that should suit your needs, but please let us know if you need further features.

```
npm i apca-w3
```
